### PR TITLE
IDEM-13

### PR DESCRIPTION
**Pull request by jrocca82**

### CODE REVIEW

File: app/src/types/mneumonic.ts [hunk 1]
```diff
@@ -0,0 +1,3 @@
+export type PrivateKey = {
+    privateKey: string;
+  }
```
Comment (rellfy) [inline app/src/types/mneumonic.ts:3]:
```suggestion
export type PrivateKey = {
    privateKey: string;
}
```
Indentation

File: app/src/utils/mneumonic-utils.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+  } catch (error) {
+    console.log(error);
+  }
+};
+
+export const getMneumonic = async () => {
+    try {
+      const mneumonic = await AsyncStorage.getItem("PRIVATE_KEY");
+      if (!mneumonic) {
+        createMneumonic()
```
Comment (rellfy) [inline app/src/utils/mneumonic-utils.ts:28]:
```suggestion
        await createMneumonic()
```

**Comment by jrocca82** [comment]
No front end but mnemonic shows in console when first navigating to Home Screen:
Screenshot 2022-05-30 121027

File: app/src/navigation/screens/Home.tsx [hunk 3]
```diff
@@ -17,6 +18,13 @@ const Home: React.FC = () => {
     navigation.navigate("Claim", { claimType });
   };
 
+  const { mnemonic, createMnemonic } = useMnemonic();
+
+  React.useLayoutEffect(() => {
+    createMnemonic();
+    console.log(mnemonic);
```
Comment (bitcoinbrisbane) [inline app/src/navigation/screens/Home.tsx:25]: Lets remove

**Review by bitcoinbrisbane — DISMISSED** [review]

File: app/src/navigation/screens/Home.tsx (hunk 3 above)
Comment (jrocca82) [reply app/src/navigation/screens/Home.tsx:25]: Leave in for testers to see it exists

File: app/src/context/Mnemonic.tsx [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+
+  const createMnemonic = async () => {
+    try {
+      const wallet = ethers.Wallet.createRandom();
+      await AsyncStorage.setItem("PRIVATE_KEY", wallet.privateKey);
+      setMnemonic(wallet.mnemonic.phrase);
+      console.log(mnemonic)
+      return mnemonic;
+    } catch (error) {
+      console.log(error);
```
Comment (JasminFarhi) [inline app/src/context/Mnemonic.tsx:42]: What about this one also @jrocca82

Comment (jrocca82) [reply app/src/context/Mnemonic.tsx:42]: No, need to catch for error, this is standard for a try/catch function

**Review by JasminFarhi — APPROVED** [review]

**Comment by jrocca82** [comment]
Screenshot 2022-05-30 153932

**Review by rellfy — APPROVED** [review]
:tada:

**Pull request merged**